lucien-ios/Dashboard/LendingViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  LendingViewController.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/9/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+final class LendingViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate {
+
+    @IBOutlet weak var collectionView: UICollectionView!
+
+    let reuseIdentifier = "comicCell"
+    let objects = ["Princess", "Gracie", "Kittens"]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        collectionView.register(UINib(nibName:"LendingCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: reuseIdentifier)
```

Comment:
String shouldn't be hardcoded. Instead, use `String(describing: LendingCollectionViewCell.self)`